src/Markdig/Parsers/OrderedListItemParser.cs

```diff
@@ -37,6 +37,7 @@ protected bool TryParseDelimiter(BlockProcessor state, out char orderedDelimiter
             {
                 if (delimiter == orderedDelimiter)
                 {
+                    state.NextChar();
                     state.NextChar();
                     return true;
                 }
```

Comment:
What is this skipping? Add a comment if necessary

Reply:
removed